scripts/set_xtensa_params.sh

```diff
@@ -81,4 +81,11 @@ case "$platform" in
 	HOST="xtensa-mt8195-elf"
 	XTENSA_TOOLS_VERSION="RI-2019.1-linux"
 	;;
+    mtl)
+	PLATFORM="mtl"
+	XTENSA_CORE="ace10_LX7HiFi4_2022_10"
+	XTENSA_TOOLS_VERSION="RI-2022.10-linux"
+	COMPILER="xt-clang"
+	LDEXTRAFLAGS=
+	;;
```

Comment:
@marc-hb Do you have ideas how to clean-up set_xtensa_params.sh. I guess this was duplicated from zephyr build when it was still a shell script.

Reply:
I think there's a lot of unused environment variables like HOST. Or at least MTL built without setting it, but maybe it's for gcc crosscompiler for other platforms. Currently testbench build is not supporting it.